### PR TITLE
Add docker-compose command

### PR DESCRIPTION
**Pull request by dannymidnight**
Simplify running the local web UI with a docker-compose command.

cc @mbelton-buildkite

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -38,9 +38,8 @@ steps:
 
 Buildkite Compat can also be used via a HTTP API using `rackup` from the `app` folder of this repository.
 
-Note: If you are using the docker image you will have to override the entrypoint:
-```shell
-$ docker run --rm -ti -p 9292:9292 --entrypoint '' --workdir /app $IMAGE:$TAG rackup --port 9292
+```sh
+docker-compose up app
```
Comment (toote) [inline Readme.md:42]: I would add it as an option to run the project but wouldn't replace the raw `docker` command.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  app:
```
Comment (toote) [inline docker-compose.yml:3]: I would change `app` to `webui` or something that would make it clear what you are running.

File: Readme.md (hunk 1 above)
Comment (toote) [inline Readme.md:42]: shouldn't we be defaulting to the plugin-version of docker compose?
```suggestion
docker compose up app
```

**Comment by dannymidnight** [comment]
How's that look @toote?

**Review by toote — APPROVED** [review]

**Pull request merged**